_config_local.yml

```diff
@@ -0,0 +1,25 @@
+theme: "just-the-docs"
```

Comment:
is it possible to move everything related to site to /docs and let docs have all of them. you have moved everything but i am not sure if we can move these config files too ?

Reply:
Yes that should be possible , will do

Reply:
this is done , moved to docs folder